frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -55,15 +56,34 @@ type Props = {| ...OwnProps, ...StateProps |};
 
 type TabKey = "data" | "general" | "defaultConfig";
 
+const AppliedSuggestionsEnum = {
+  Yes: "Yes",
+  No: "No",
+  NoAvailableSuggestions: "NoAvailableSuggestions",
+};
+
+const IsJSONFormatValidEnum = {
+  Yes: "Yes",
+  No: "No",
+  BrokenFormat: "BrokenFormat",
```

Comment:
I would still opt to rename this to BrokenJson

Reply:
Sorry I mix this up when I added the enums :see_no_evil: